README.md

```diff
@@ -182,32 +182,35 @@ Responses:
 
 Query parameters:
 
-* `cityId` Integer. If specified, returns only teams based in the city with given id.
-* `cityName` String. If specified, returns only teams based in the city with given name.
+* `city` Integer. If specified, returns only teams based in the city with given id.
```

Comment:
I think it's better to explicitly use `cityId` because then it's clear already from the parameter name that it refers to city's id. Also then it doesn't require client changes if they were already using `cityId`